tailwind.config.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
           }
           return `rgb(var(--green))`
         },
+        red: '#BC0E13',
```

Comment:
I think a better approach would be to define this color in `styles/global.css` and use it to define color the way `back` and `green` are defined preceding it.

This will allow us to use an opacity filter along with color in case we need it, and will also maintain symmetry of how we have defined custom colors till now.

Reply:
nice catch, I will add all the custom colors used in this challenge to tailwind config file as they might be re-used in future challenges.

Reply:
This comment was left to be resolved in the subsequent updates of PR.

Since it's not a blocker, let's take this up with #454.